File: A2_Q2.c
/*
Create two vectors of type A=(a1,a2,a3) and B=(b1,b2,b3) with numbers of
your choice (butnot random numbers) in the code itself. Find A+B and A.B 
(dot product).
*/
#include<stdio.h>
#include<stdlib.h>

int main()
{
	float A[]={4,-3.5,5};
	float B[]={2.5,6,0};
	float sum[]={0,0,0};
	float dot_pdt=0;
	int i,j;
	
	for(i=0;i<=2;i++)
	{
		sum[i]=A[i]+B[i];
	}
	for(j=0;j<=2;j++)
	{
		dot_pdt=dot_pdt+A[j]*B[j];
	}
	printf("\nThe addition A+B vecter is : ");
	for(i=0;i<=2;i++)
	{
		printf("%f  ",sum[i]);
	}

	printf("\n\nThe dot product A.B is :  %f\n",(float)dot_pdt);
	
	return 0;
}

File: A2_Q2.py
"""
Create two vectors of type A=(a1,a2,a3) and B=(b1,b2,b3) with numbers of
your choice (butnot random numbers) in the code itself.
Find A+B and A.B (dot product).
"""
A=[-2.5,5,6]
B=[6,0,-1]
sum=[0]*3
dot_pdt=0
#addition of two vecter(A+B)
for i in range(len(A)):
    sum[i]+=A[i]+B[i]
print("\nThe addition A+B vecter is : ",sum,"\n")
#dot product of two vecter
for j in range(len(A)):
    dot_pdt+=A[j]*B[j]
print("The dot product A.B is : ",dot_pdt)

